tools/cli/config.go

```diff
@@ -0,0 +1,114 @@
+// The MIT License
```

Comment:
this file is a copy of https://github.com/temporalio/tctl/blob/main/cli/config.go